notebooks/haystack-qa.ipynb

```diff
@@ -0,0 +1,1462 @@
+{
```

Comment:
<p>One way to inspect the model would be to observe what retriever is fetching. Maybe it's not providing enough context to the generator. What happens if we tweak retriever model parameters?</p><p> </p><p>Also. another possible explanation could be that these embedding models may not be advanced enough to capture language constructs leading to poor answers.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>